src/js/outreach/apps/verify_app.js

```diff
@@ -20,8 +20,6 @@ export default App.extend({
     return getPatientInfo({ actionId: this.actionId });
   },
   onFail(options, response) {
-    if (response.status >= 500) return;
```

Comment:
This handles errors on the `GET /api/outreach?filter[action]=` request.

If that request `500`s, this `onFail()` doesn’t seem to get called. It’s handled directly by the error app. So I believe this will always be `false` and therefore not needed.